documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,33 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting interval between precopies
+
+You can set the precopy interval.
+
+.Handling RHV VMs with USB enabled
+
+You can migrate an RHV VM with a host device of the USB type mapped to it, although the mapping is not supported on the target side.
+
+[id="new-features-and-enhancements-22_{context}"]
+== New features and enhancements
+
+This release adds the following features and improvements.
+
+.Downloading must-gather logs from the web console
+
+You can download logs for a migration plan or a migrated VM in the {project-full} web console.
+
+.Cloning a plan to use as a new plan
+
+You can make a copy of an existing migration plan and use the copy as a new plan.
+
+.Archiving a plan from the UI
+
+You can archive a migration plan that is no longer in use. Archived plans can be viewed and duplicated but cannot be edited.
```

Comment:
Change to: "You can archive a migration plan. Archived plans can be viewed or cloned. They cannot be edited or unarchived."